docs/advanced_usage.md

```diff
@@ -113,6 +113,62 @@ Yubikey support requires
 (which are bundled with the PIV tools)</a> to be available in standard
 library locations.
 
+### Use the native store
```

Comment:
Awesome docs! 👍 

I'm thinking it should be clear that this is for keys only, so it might be worth renaming the section to "Use the native key store"?